gfw_pixetl/utils/gdal.py

```diff
@@ -137,25 +137,25 @@ def get_metadata(
             blockysize=band["block"][1],
         )
 
-        try:
-            if compute_stats:
+        if compute_stats:
```

Comment:
Should exceptions still be caught if they're not KeyErrors? Also should this KeyError log something, or is there nothing worth logging?

Reply:
No, I think if they're KeyErrors we can swallow them safely but it's not worth logging because it's apparently normal to have empty tiles for which computing stats doesn't make any sense. Also, the log never makes it out of the process unless there's an uncaught exception as things are. As for other Exceptions, I think we want them to bubble-up and kill the process, in which case whatever that exception is will be logged.